### ISSUE TITLE
Add swipe gestures to enable it to play in mobile

### ISSUE THREAD

**Issue by SubhamKarmakar24** [SubhamKarmakar24/snake#7]
For now, only the desktop version works. Mobile version to be implemented using swipe gestures

**Comment by AshAman999** [comment]
can do so but will need time till tommorow ,cuz broh you have defined movement in different files for different game   modes@SubhamKarmakar24 ,
if you can wait then assign it to me i can work on that when i'm free
Anyway it is really cool.💙💙

**Comment by SubhamKarmakar24** [comment]
> 
> 
> can do so but will need time till tommorow ,cuz broh you have defined movement in different files for different game modes@SubhamKarmakar24 ,
> if you can wait then assign it to me i can work on that when i'm free
> Anyway it is really cool.💙💙

Yeah, take your time. You can change one file and we will wait for the issue #3 to be resolved so that it works for all game modes

**Comment by AshAman999** [comment]
@SubhamKarmakar24  Hello broh, i have made changes in the limitless1.html page,as i told you earlier there is seperate script for every game mode,so i will have to make changes in all of em,
kinda busy with homework now, would work with rest of pages tommorow.
So should i make a pr for the work till now?

**Comment by AshAman999** [comment]
here is the pr i made, 
[solution](https://github.com/SubhamKarmakar24/snake/pull/14)

**Comment by SubhamKarmakar24** [comment]
> 
> 
> here is the pr i made,
> [solution](https://github.com/SubhamKarmakar24/snake/pull/14)

If you plan to add touch controls for the last 2 modes, let me know. The different game modes have been combined

**Comment by AshAman999** [comment]
My classes are going on so little busy, i will make a pr soon

and thanks for merging sir

**Comment by SubhamKarmakar24** [comment]
> 
> 
> My classes are going on so little busy, i will make a pr soon
> 
> and thanks for merging sir

Yeah. Take time. Everyone's busy with the online classes these days : )
